product-service/serverless.ts

```diff
@@ -60,6 +65,18 @@ const serverlessConfiguration: Serverless = {
         }
       ]
     },
+    createProduct: {
+      handler: 'handler.createProduct',
+      events: [
+        {
+          http: {
+            method: 'post',
+            path: 'products',
+            cors: true
```

Comment:
Or you can just use httpApi to not include cors: true into every event
provider:
  httpApi:
    cors: true

Reply:
Cool. Thanks. Added to provider configuration